docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
       - "5000:5000"
     volumes:
       - ".:/app"
+      - /app/client/dist
```

Comment:
What's this for?

Reply:
it needs for /dist created by `npm run build`. Please refer to **The node_modules Volume Trick** chapter on https://jdlm.info/articles/2016/03/06/lessons-building-node-app-docker.html

Reply:
The way this `docker-compose.yml` file was designed to work is with the user running npm related stuff on the host, so `client/dist` will be part of the mounted volume.

See: https://redash.io/help/open-source/dev-guide/docker for details.

Reply:
well.. we seem to need more hack for this point 🤔

Reply:
Wait, I just realized /dist created by `npm run build` in Dockerfile will be overwritten by volume `.:/app`. So, `npm run build` is not necessary under development, right? 
Btw, is this Dockerfile used in production? (IOW, is this Dockerfile of redash/redash?)

Reply:
> So, npm run build is not necessary under development, right?

Correct.